lib/pages/main_menu.dart

```diff
@@ -170,6 +170,224 @@ class _MainMenuState extends State<MainMenu> {
                                     child: Text('Sources',
                                         style: new TextStyle(color: purple.shade500, fontSize: 20))
                                 ),
+                                ListView(
+                                  shrinkWrap: true,
+                                  children: <Widget>[
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
```

Comment:
I do not think that this icon makes sense, as it does not help to indicate the separation between sources, and as it is the same icon for every list item, it does not really convey any more information to the user. I would remove the bullets/icons and add a divider between list items to visually divide them.